pkg/api/vlabs/validate_test.go

```diff
@@ -5272,7 +5272,7 @@ func TestValidateAzureStackSupport(t *testing.T) {
 			cs.Properties.OrchestratorProfile.OrchestratorVersion = "1.16.11"
 			if err := cs.Validate(false); !helpers.EqualError(err, test.expectedErr) {
 				t.Logf("scenario %q", test.name)
-				t.Errorf("expected error: %v, got: %v", test.expectedErr, err)
+				t.Logf("FIXME: expected error: %v, got: %v", test.expectedErr, err)
```

Comment:
@jadarsie if I follow our usual process, this leaves Azure Stack without a supported 1.16.x version, so I disabled this test. But let me know if you prefer a different approach--we can be flexible about version support if need be.

Reply:
ASH support for v1.16 here #3613, usual process OK

Reply:
@jadarsie should I merge this as-is, and then #3613 can rebase and undo the error log changes?

Reply:
@JesusAlvarezTorres FYI

Reply:
@jackfrancis go ahead and merge this in, I'll update #3613 when this is in